### ISSUE TITLE
negation

### ISSUE THREAD

**Issue by avillafiorita** [sds/propose#1]
Hi 
I tried the following: (p or not p), but it seems that the two instances of p are treated like two distinct propositional letters:

```
propose
> p or not p
+---+---+--------+
|     p ∨ ¬p     |
+---+---+--------+
| p | p | Result |
+---+---+--------+
| T | T |   T    |
| F | T |   F    |
| T | F |   T    |
| F | F |   T    |
+---+---+--------+
```

**Comment by sds** [comment]
Thanks for reporting this! Rolled out a fix in [1.0.1](https://github.com/sds/propose/releases/tag/v1.0.1).

Here's the output now:

```
⨠ propose
> p or not p
+---+--------+
|   p ∨ ¬p   |
+---+--------+
| p | Result |
+---+--------+
| T |   T    |
| F |   T    |
+---+--------+
```

**Comment by avillafiorita** [comment]
Thank you very much, really appreciated and super-fast fix.
